src/hotspot/share/opto/superword.cpp

```diff
@@ -1806,7 +1941,7 @@ void SuperWord::filter_packs_for_profitable() {
   while (true) {
     int old_packset_length = _packset.length();
     filter_packs(nullptr, // don't dump each time
-                 "size is not a power of 2",
+                 "not profitable",
```

Comment:
Note: this was a mistake of a previous PR. Only relevant for debug printing, so I just fix it here.